app/common/state/bookmarksState.js

```diff
@@ -187,7 +187,9 @@ const bookmarksState = {
     }
 
     siteKeys.forEach((siteKey) => {
-      state = state.setIn([STATE_SITES.BOOKMARKS, siteKey, 'favicon'], favicon)
+      if (state.getIn([STATE_SITES.BOOKMARKS, siteKey])) {
+        state = state.setIn([STATE_SITES.BOOKMARKS, siteKey, 'favicon'], favicon)
```

Comment:
this change in particular was causing @darkdh to have duplicate entries... one regular and one with only a favicon entry